lib/data_clean_utils.py

```diff
@@ -6,19 +6,10 @@
     Cleans up all strings from each column of the NYC Taxi dataset.
 """
 def hard_coded_cleaning_steps_mutation(df: pd.DataFrame):
-    # Correct string type confusion in vendorid.
-    df.loc[df['vendorid'] == '1', 'vendorid'] = 1
-    df.loc[df['vendorid'] == '1.0', 'vendorid'] = 1
-    df.loc[df['vendorid'] == '2', 'vendorid'] = 2
-    df.loc[df['vendorid'] == '2.0', 'vendorid'] = 2
+    # Correct string type confusion in vendorid, payment_type
+    df['vendorid'] = df['vendorid'].astype(float).astype(int)
```

Comment:
Why `astype(float)` before `astype(int)`?

Reply:
I needed to massage the type conversions because int("1.0") is invalid but int(float("1.0")) is valid.